src/Components/Server/ref/Microsoft.AspNetCore.Components.Server.netcoreapp3.0.cs

```diff
@@ -21,7 +21,7 @@ public static partial class ComponentEndpointRouteBuilderExtensions
         public static Microsoft.AspNetCore.Components.Server.ComponentEndpointConventionBuilder MapBlazorHub<TComponent>(this Microsoft.AspNetCore.Routing.IEndpointRouteBuilder endpoints, string selector, string path, System.Action<Microsoft.AspNetCore.Http.Connections.HttpConnectionDispatcherOptions> configureOptions) where TComponent : Microsoft.AspNetCore.Components.IComponent { throw null; }
     }
 }
-namespace Microsoft.AspNetCore.Components.Browser.Rendering
+namespace Microsoft.AspNetCore.Components.Web.Rendering
```

Comment:
Note: there's a bunch of places where the `.Browser` namespace is used outside of this library/package. I'm not going through these with a fine-toothed cone at this point, because these will be reviewed as well. I'm trying to get the package rename out of the way right now.